apps/web/src/locales/en/messages.json

```
@@ -177,7 +177,7 @@
     "message": "Total applications"
   },
   "unmarkAsCriticalApp": {
-    "message": "Unmark as critical app"
+    "message": "Unmark as critical"
```

```suggestion
"unmarkAsCritical": {
    "message": "Unmark as critical"
```

This suggestion shows how the key also needs to change, but also requires changing all usages.
Just as an example: https://github.com/bitwarden/clients/blob/5f7f1d19242c418bb7f989ce50dc0c528e27b22c/bitwarden_license/bit-web/src/app/dirt/access-intelligence/app-table-row-scrollable.component.html#L92